projects/miopen/test/gtest/solver_bwd.hpp

```
@@ -63,7 +63,7 @@ struct ConvBwdSolverTest
         {
             // Do not put GTEST_SKIP here.
             // The usage of non-applicable config should be considered as a bug in the test.
```

Maybe instead of a comment, just put this message directly into the gtest failure message.  Same for all the other cases below.

---

How about `MIOPEN_GTEST_FAIL("xyz Solver is not applicable. Invalid configs cannot be tested.");`?

It should actually throw an exception or some other trappable mechanism so we can add tests for expected failures. But I think that's work for the _very_ distant future so I'm not even bothering to add Issues for such things.